src/utils.ts

```diff
@@ -0,0 +1,10 @@
+export const getCurrentTabUrl = (
```

Comment:
does this have to be a callback or can it just return the current tab url as string?